dsmr_parser/parsers.py

```diff
@@ -96,6 +96,7 @@ def parse(self, telegram_data, encryption_key="", authentication_key=""):  # noq
                 except Exception:
                     logger.error("ignore line with signature {}, because parsing failed.".format(signature),
                                  exc_info=True)
+                    raise
```

Comment:
I think this is ok, but only if all the [clients](https://github.com/ndokter/dsmr_parser/tree/master/dsmr_parser/clients) that use the TelegramParser will handle the error and keep working after encountering an error. What do you think?

Reply:
Was this a reaction to my comment?
If yes, then I do not understand it, because if you would do it like i proposed, nothing changes for existing clients. The default behaviour would be to not propagate the parse exception.